migration/wcmpbe-upgrade-migration-v4-3-6.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+use migration\WCMPBE_Upgrade_Migration;
+
+if (! defined('ABSPATH')) {
+    exit;
+}
+
+if (class_exists('WCMPBE_Upgrade_Migration_v4_3_6')) {
+    return new WCMPBE_Upgrade_Migration_v4_3_6();
+}
+
+/**
+ * Migrates pre v4.3.6 settings
```

Comment:
Misschien hier even de changes opsommen, anders moeten we de codes in de toekomst weer gaan ontcijferen